pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1461,6 +1461,7 @@ func PublishAppInfoToZedCloud(ctx *zedagentContext, uuid string,
 				networkInfo := getNetInfo(interfaceDetail, false)
 				ip, allocated, macAddr := getAppIP(ctx, aiStatus,
 					ifname)
+				log.Infof("PublishAppInfoToZedCloud: after getAppIP, ifname %s, ip %s, mac %s", ifname, ip, macAddr)
```

Comment:
This is in Handle metrics.. Getting sent repeatedly.. Why do u need it to be an Info message? Is it just needed only for debugging when developing? We anyway get to see the value in the Status sent to the cloud?

Reply:
sure. will remove this info